.concourse/kubecf-pool-reconciler.yaml.gomplate

```diff
@@ -0,0 +1,69 @@
+{{ $kind_pool_size := 2 }}
+
+resources:
+- name: kind-environments
+  type: pool
+  source:
+    uri: git@github.com:SUSE/cf-ci-pools.git
+    branch: kubecf-kind-pools
+    pool: kind
+    private_key: ((github-private-key))
+
+- name: catapult
+  type: git
+  source:
+    uri: https://github.com/SUSE/catapult
+  version:
+    ref: 543bc71abc8bd22e5ef81fdf94a7afeb2f99066d
+
+
+deploy_args: &deploy_args
+- -ce
+- |
```

Comment:
We should be moving away from embedding entire bash scripts into the pipeline file; tasks are capable of reading these from external files in a git repo (this one) and the commit ID used at the time the pipeline was last deployed/updated can be used to ensure the script doesn't change unless the pipeline does

Reply:
We have a separate story for this: https://github.com/cloudfoundry-incubator/kubecf/issues/443 . It would be out of scope for this PR